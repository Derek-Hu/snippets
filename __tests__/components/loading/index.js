import React from 'react';
import Loading from '~/components/loading/index';

it('Loading样式', () => {
  const wrapper = mount(<Loading />);

  expect(wrapper).toMatchSnapshot();
});
