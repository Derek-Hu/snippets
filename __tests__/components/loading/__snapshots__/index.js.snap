// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading样式 1`] = `
<Loding>
  <div
    className="spin-container"
  >
    <Spin
      size="default"
      spinning={true}
      wrapperClassName=""
    >
      <div
        className="ant-spin ant-spin-spinning"
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </Spin>
  </div>
</Loding>
`;
